src/components/shared/Addresses.js

```diff
@@ -0,0 +1,159 @@
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+import { getAddresses, postAddress } from "../../services/Cabecudos";
+import Address from "./Address";
+import { GrFormClose } from 'react-icons/gr';
+import { useContext } from "react/cjs/react.development";
```

Comment:
Acontece², sempre teste o deploy do PR pra ver se ta tudo ok
```suggestion
import { useContext } from "react";
```